mozci/console/commands/push.py

```diff
@@ -946,6 +955,107 @@ def list_classification_tasks(self, group_id):
+        except ValueError:
+            self.line("<error>Provided --nb-pushes should be an int.</error>")
+            exit(1)
+
+        self.line("<comment>Loading pushes...</comment>")
+        self.pushes = make_push_objects(nb=nb_pushes, branch=branch)
+
+        for push in self.pushes:
+            backfill_tasks = []
+
+            try:
+                indexed_tasks = list_indexed_tasks(
+                    f"gecko.v2.{push.branch}.revision.{push.rev}.taskgraph.actions"
+                )
+            except requests.exceptions.HTTPError as e:
+                self.line(
+                    f"<error>Couldn't fetch indexed tasks on push {push.push_uuid}: {e}</error>"
+                )
+                continue
+
+            for indexed_task in indexed_tasks:
+                task_id = indexed_task["taskId"]
+                try:
+                    children_tasks = list_dependent_tasks(task_id)
+                except requests.exceptions.HTTPError as e:
+                    self.line(
+                        f"<error>Couldn't fetch dependent tasks of indexed task {task_id} on push {push.push_uuid}: {e}</error>"
+                    )
+                    continue
+
+                for child_task in children_tasks:
+                    task_action = (
+                        child_task.get("task", {}).get("tags", {}).get("action", "")
+                    )
+                    th_symbol = (
+                        child_task.get("task", {})
+                        .get("extra", {})
+                        .get("treeherder", {})
+                        .get("symbol", "")
+                    )
+                    if task_action == "backfill-task" and th_symbol.endswith("-bk"):
+                        status = child_task.get("status", {})
+                        backfill_tasks.append(
+                            {
+                                "taskId": status.get("taskId"),
+                                "taskGroupId": status.get("taskGroupId"),
+                                "state": status.get("state"),
+                            }
+                        )
+
+            def group_key(task):
+                return task["taskGroupId"]
+
+            backfill_tasks = sorted(backfill_tasks, key=group_key)
+            for group_id, value in groupby(backfill_tasks, group_key):
+                tasks = list(value)
+                if all(task["state"] == "completed" for task in tasks):
+                    index_path = f"project.mozci.check-backfill.{environment}.{push.branch}.{push.rev}.{group_id}"
+                    already_indexed = list_indexed_tasks(
+                        index_path, root_url=COMMUNITY_TASKCLUSTER_ROOT_URL
+                    )
+                    if already_indexed:
+                        continue
+
+                    # Populating the index with the current task to prevent sending the following notification once again
+                    insert_task(
```

Comment:
Only run that when TASK_ID is available (it will only be available on Taskcluster, not while running locally)